wp-content/themes/core/components/blocks/quote/quote.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Tribe\Project\Templates\Components\blocks\quote\Quote_Block_Controller;
+
+/**
+ * @var array $args Arguments passed to the template
+ */
+// phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+$c = Quote_Block_Controller::factory( $args ); ?>
+
+<section <?php echo $c->classes(); ?> <?php echo $c->attrs(); ?>>
+	<div <?php echo $c->container_classes(); ?>>
+
+		<?php if ( $c->media ) { ?>
+			<div <?php echo $c->media_classes(); ?>>
+				<?php get_template_part(
```

Comment:
Same q here re: wrapped arguments for `get_template_part()`

Reply:
In this instance it's like 87 characters. In the others, it's out of habit.